### PR TITLE
add tests to equalTo with more than one condition and with the same key

### PR DESCRIPTION
**Pull request by igor-mobapps**
tests for the problem listed here  #476

### CODE REVIEW

**Comment by dplewis** [comment]
@igor-mobapps Thanks for the test cases? Can you update from master so the build can pass?

I tried your tests locally and they passed.

File: tests/Parse/ParseQueryTest.php [hunk 1]
```diff
@@ -2720,4 +2720,56 @@ public function testUnknownCondition()
...
+                'status' => 'qux'
+            ]
+        ]);
+        $qux->save();
+
+        $query = new ParseQuery('TestObject');
+        $query->notEqualTo('fooStack.status', 'baz');
+        $query->equalTo('fooStack.status', 'bar');
+
+        $this->assertEquals(3, $query->count(true));
```
Comment (dplewis) [inline tests/Parse/ParseQueryTest.php:2760]: Is this the expected results? Should these test fail instead of passing?

Comment (igor-mobapps) [reply tests/Parse/ParseQueryTest.php:2760]: yes, they must fail. I just ran the tests, but after fixing it with the solution mentioned in issue #476 , they should fail

**Comment by dplewis** [comment]
Closing via https://github.com/parse-community/parse-php-sdk/pull/510

**Pull request closed without merging**